modules/logging-content-filter-drop-records.adoc

```diff
@@ -6,103 +6,122 @@
 [id="logging-content-filter-drop-records_{context}"]
 = Configuring content filters to drop unwanted log records
 
-When the `drop` filter is configured, the log collector evaluates log streams according to the filters before forwarding. The collector drops unwanted log records that match the specified configuration.
+Collecting all cluster logs produces a large amount of data, which can be expensive to move and store. To reduce volume, you can configure the `drop` filter to exclude unwanted log records before forwarding. The log collector evaluates log streams against the filter and drops records that match specified conditions.
+
+The `drop` filter uses the `test` field to define one or more conditions for evaluating log records.  
+The filter applies the following rules to check whether to drop a record:
+
+* If a test passes, the filter drops the log record.
+* A test passes if all its specified conditions evaluate to true.
+* If you define several tests in the `drop` filter configuration, the filter drops the log record if any of the tests pass.
+* If there is an error evaluating a condition, for example, the referenced field is missing, that condition evaluates to false.
 
 .Prerequisites
 
 * You have installed the {clo}.
 * You have administrator permissions.
 * You have created a `ClusterLogForwarder` custom resource (CR).
+* You have installed the {oc-first}.
 
 .Procedure
 
-. Add a configuration for a filter to the `filters` spec in the `ClusterLogForwarder` CR.
+. Extract the existing `ClusterLogForwarder` configuration and save it as a local file.
++
+[source,terminal]
+----
+$ oc get clusterlogforwarder <name> -n <namespace> -o yaml > <filename>.yaml
+----
++
+Where:
 +
-The following example shows how to configure the `ClusterLogForwarder` CR to drop log records based on regular expressions:
+* `<name>` is the name of the `ClusterLogForwarder` instance you want to configure.
+* `<namespace>` is the namespace where you created the `ClusterLogForwarder` instance, for example `openshift-logging`.
+* `<filename>` is the name of the local file where you save the configuration.
+
+. Add a configuration to drop unwanted log records to the `filters` spec in the `ClusterLogForwarder` CR.
 +
+--
 .Example `ClusterLogForwarder` CR
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-# ...
+  name: instance
+  namespace: openshift-logging
 spec:
+  # ...
   filters:
-  - name: <filter_name>
+  - name: drop-filter
     type: drop # <1>
     drop: # <2>
     - test: # <3>
-      - field: .kubernetes.labels."foo-bar/baz" # <4>
+      - field: .kubernetes.labels."app.version-1.2/beta" # <4>
         matches: .+ # <5>
       - field: .kubernetes.pod_name
         notMatches: "my-pod" # <6>
   pipelines:
-  - name: <pipeline_name> # <7>
-    filterRefs: ["<filter_name>"]
-# ...
+  - name: my-pipeline # <7>
+    filterRefs: ["drop-filter"]
```

Comment:
minor comment:    `filterRefs` is an array (same as inputRef and outputRef), so the brackets are another way of expressing the list of filters here.   For consistency, we may consider updating this to match the "listed" approach as used in other examples:
```
- name: my-pipeline
  filterRefs:
  - drop-filter
```